integration-tests/send-email.integration.test.ts

```diff
@@ -0,0 +1,375 @@
+/**
+ * Integration test for send email functionality
+ *
+ * This test verifies the email sending functionality by making actual API calls
+ * to the Dynatrace environment. These tests require valid authentication credentials
+ * and the email:emails:send scope.
+ */
+
+import { config } from 'dotenv';
+import { createDtHttpClient } from '../src/authentication/dynatrace-clients';
+import { sendEmail, EmailRequest } from '../src/capabilities/send-email';
+import { getDynatraceEnv, DynatraceEnv } from '../src/getDynatraceEnv';
+
+// Load environment variables
+config();
+
+const API_RATE_LIMIT_DELAY = 100; // Delay in milliseconds to avoid hitting API rate limits
+
+const scopesBase = [
+  'app-engine:apps:run', // needed for environmentInformationClient
+  'app-engine:functions:run', // needed for environmentInformationClient
+];
+
+const scopesEmail = [
+  'email:emails:send', // Send emails via Dynatrace Email API
+];
+
+describe('Send Email Integration Tests', () => {
+  let dynatraceEnv: DynatraceEnv;
+
+  // Setup that runs once before all tests
+  beforeAll(async () => {
+    try {
+      dynatraceEnv = getDynatraceEnv();
+      console.log(`Testing against environment: ${dynatraceEnv.dtEnvironment}`);
+    } catch (err) {
+      throw new Error(`Environment configuration error: ${(err as Error).message}`);
+    }
+  });
+
+  afterEach(async () => {
+    // Add delay to avoid hitting API rate limits
+    await new Promise((resolve) => setTimeout(resolve, API_RATE_LIMIT_DELAY));
+  });
+
+  describe('Basic Email Sending', () => {
+    it('should send a simple plain text email successfully', async () => {
+      const dtClient = await createDtHttpClient(
+        dynatraceEnv.dtEnvironment,
+        scopesBase.concat(scopesEmail),
+        dynatraceEnv.oauthClientId,
+        dynatraceEnv.oauthClientSecret,
+        dynatraceEnv.dtPlatformToken,
+      );
+
+      const emailRequest: EmailRequest = {
+        toRecipients: { emailAddresses: ['test@example.com'] },
```

Comment:
Any integration-test run will now send an e-mail to `test@example.com`, potentially spamming this email address. 
I'm not happy about doing this, and I'd rather like to see those e-mails go into an inbox that we control.

Reply:
I'm all for having a test-case - right now I'm clarifying with the team responsible for the e-mail API internally on whether there is a dry-run flag or alike, which would allow us to consume the API without actually sending the e-mail.

In case we don't have that, I'd envision to make this test optional and the e-mail address configurable.